app/src/test/java/edu/nextstep/camp/calculator/MainPresenterTest.kt

```diff
@@ -631,8 +628,8 @@ class MainPresenterTest {
     @Test
     fun `32 + 수식이 입력되었을때 연산을 수행하면 유효하지 않은 수식에 대한 에러 메시지를 보여줘야 한다`() {
         // given
-        val errorMessageSlot = slot<UiText>()
-        every { view.showErrorMessage(capture(errorMessageSlot)) } answers { nothing }
+        val errorExceptionSlot = slot<Exception>()
+        every { view.showErrorMessage(capture(errorExceptionSlot)) } answers { nothing }
```

Comment:
```suggestion
        every { view.showErrorMessage(capture(errorExceptionSlot)) } just Runs
```
해당 함수의 호출 결과를 스텁할 것이 없다면, `just Runs`를 활용할 수 있어요. (이하 관련 내용 동일)